### PR TITLE
[bind9] Fix and re-enable AFL fuzzing engine

### PR DESCRIPTION
**Pull request by oerdnj**

### CODE REVIEW

**Comment by oerdnj** [comment]
The AFL fuzzing engine seems to hang the `isc_lex_getmastertoken` fuzzer quite early (which is consistent with my findings today^Hyesterday). I don't know if that poses a problem to merging this, but the BIND 9 team will try to fix this in the upstream repository, it should definitely not hang.

Again thanks, this is a great help!

**Comment by Dor1s** [comment]
Given that a fuzz target times out with AFL right away, we consider it a "startup crash" and do not archive such builds (since it's pointless to run the on fuzzing infra). When at least one fuzzing build configuration is broken, the infrastructure will report a build failure. That's why it's not recommended to enable AFL until it can pass the presubmit check -- it's useless and even harmless from OSS-Fuzz point of view.

Let's wait for the issue to be fixed in bind9, and then we'll happily re-enable AFL config for this project.

File: projects/bind9/project.yaml [hunk 1]
```diff
@@ -2,11 +2,12 @@ homepage: "https://gitlab.isc.org/isc-projects/bind9"
 language: c
 primary_contact: "security-officer@isc.org"
 auto_ccs:
-  - "ondrej@sury.org"
+  - "ondrej@isc.org"
```
Comment (Dor1s) [inline projects/bind9/project.yaml:5]: let's land this part separately as it seems important. i'll commit it directly to save time

Comment (Dor1s) [reply projects/bind9/project.yaml:5]: https://github.com/google/oss-fuzz/commit/5f1504cb10e1624b2d0098ce39c64562834cd955

**Comment by oerdnj** [comment]
Thanks for the emails update.

Anyway, we fixed the `afl` hang on startup, the fuzzer was just looping on EOF, so this can be safely merged now.

**Pull request merged**